augmented_reality/Examples/ArExample.cpp

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
I think this is our git hooks doing this, as well as some other funky stuff with calling our Headers STL headers since in toolkit it looks like we use `< ... >` instead of ` "..." `